subprojects/inline/src/test/java/org/mockitoinline/ConstructionMockTest.java

```diff
@@ -50,10 +50,11 @@ public void testConstructionMockDefaultAnswer() {
 
     @Test
     public void testConstructionMockDefaultAnswerMultiple() {
```

Comment:
Can we retain the original test (e.g. 2 invocation definitions and invoking it 3 times) and instead add a new regression test which is this new format? That way we ensure that both scenarios still work, thanks!

Reply:
Done